nats-base-client/core.ts

```diff
@@ -651,7 +651,7 @@ export interface NatsConnection {
    *
    * @param opts
    */
-  jetstream(opts?: JetStreamOptions): JetStreamClient;
+  jetstream(opts?: JetStreamOptions | JetStreamManagerOptions): JetStreamClient;
```

Comment:
Should this be `JetStreamOptions & JetstreamManagerOptions` - this would allow all options be be specified.

Reply:
Doh - the current type for JetStreamManagerOptions extends JetStreamOptions, so this is a good change.